Tests/IntegrationTest.php

```diff
@@ -27,7 +27,13 @@ public function testServiceWiring()
         $result = $container->get('public.twig')->render('@integration_test/template.twig', [
             'yesterday' => (new \DateTime('-1 day'))
         ]);
+
+        $resultFr = $container->get('public.twig')->render('@integration_test/templateSpecificLocale.twig', [
```

Comment:
I think we should move this to its own test.. e.g. `testLocalTanslation()` or something along those lines. The current `testServiceWiring()` method is is primarily a smoke test to assert that the bundles configuration is working correctly.